frontend/src/components/pages/TeamDashboard/Presentation.js

```diff
@@ -33,11 +33,18 @@ const useStyles = makeStyles((theme) => ({
   },
 }));
 
-const ReviewLineChart = ({ data, minimum, maximum }) => {
-  // TODO calculate maximum and minimum so that all the graphs have the same proportions
-
+const ReviewLineChart = ({ data, allUsersData }) => {
+  const reviewsArr = []
```

Comment:
You have been adding semicolons at the end of each and every statement, but not here, please be consistent